tests/rpm/test_rpm.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Product tests for generating an online and offline installer for presto-admin
+"""
+
+from tests.no_hadoop_bare_image_provider import NoHadoopBareImageProvider
+from tests.product.base_product_case import BaseProductTestCase, docker_only
+from tests.product.standalone.presto_installer import StandalonePrestoInstaller
+
+
+class TestRpm(BaseProductTestCase):
+    def setUp(self):
+
+        super(TestRpm, self).setUp()
+        self.setup_cluster(NoHadoopBareImageProvider(), self.PA_ONLY_CLUSTER)
+
+    @docker_only
```

Comment:
hm, if most of these are going to be docker_only I'd think it would be ok to not have the ability to run rpm tests on a cluster.